### PR TITLE
Add a Form for Adding Meals to List

### PR DESCRIPTION
**Pull request by lkk95**
In this PR I created a form which adds a new meal to the list shown on Homepage.
Input validation is not yet implemented, this will come in a later user story when I implemented all input fields.

Testing was moved to another US as well.

### CODE REVIEW

File: src/components/CreateForm/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import useStore from '../../hooks/useStore.js';
+import Button from '../Button/formbutton.js';
+
+import FormContainer from './styled.js';
+
+export default function CreateForm() {
+	const [title, setTitle] = useState('');
+	const [category, setCategory] = useState('');
+	const [isChecked, setChecked] = useState(true);
```
Comment (MathiasHobler) [inline src/components/CreateForm/index.js:11]: Hi Laura, you use two different states for one Object, I would combine them into one state like this: const [newMeal, setNewMeal] = useState(
{
tittle: '',
category: ''
}
)

File: src/components/CreateForm/styled.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+const FormContainer = styled.form`
```
Comment (isabellpaulmann) [inline src/components/CreateForm/styled.js:3]: Maybe its better to name the FormContainer > StyledFormContainer, so you know that this is a styled component :)

File: src/components/CreateForm/index.js (hunk 1 above)
Comment (lkk95) [reply src/components/CreateForm/index.js:10]: Thank you, Mathias, for your review. :)
I combined the two states now into one.

**Review by MathiasHobler — APPROVED** [review]
lgtm

**Pull request merged**